internal/domain/quizzes/quiz_repository.go

```diff
@@ -654,3 +653,26 @@ func (a *QuizRepository) InsertQuestionAnswer(ctx context.Context, questionAnswe
 
 	return nil
 }
+
+func (a *QuizRepository) Delete(ctx context.Context) error {
+	// Prepare the DELETE statement
+	query := `DELETE FROM quizzes WHERE id = $1`
+
+	stmt, err := a.db.PrepareContext(ctx, query)
+	if err != nil {
+		return status.Errorf(codes.Internal, "Prepare statement delete quizzes: %v", err)
+	}
+	defer stmt.Close()
+
+	result, err := stmt.ExecContext(ctx, a.pb.Id)
+	if err != nil {
+		return status.Errorf(codes.Internal, "Exec delete quizzes: %v", err)
+	}
+
+	_, err = result.RowsAffected()
+	if err != nil {
+		return status.Errorf(codes.Internal, "Error getting rows affected: %v", err)
+	}
+
```

Comment:
ini sebelumnya ada id not found ya mas @indrafani03 ?
dikembalikan saja mas, biar kelihatan ada record yang dihapus apa noga.

```
affected, err := rs.RowsAffected()
	if err != nil {
		return status.Errorf(codes.Internal, "Error getting rows affected: %v", err)
	}

	if affected == 0 {
		return status.Errorf(codes.InvalidArgument, "id not found: %v", err)
	}
```

Reply:
ok baik mas

Reply:
sudah saya tambahin lagi mas